From 779428ad7d180f8d4a72282384530ee543974623 Mon Sep 17 00:00:00 2001
From: Elaina Thompson <felinae@ulthar.cat>
Date: Sun, 23 Jan 2022 08:28:49 -0800
Subject: [PATCH] rip out /x/

---
 main.go | 34 ----------------------------------
 1 file changed, 34 deletions(-)

diff --git a/main.go b/main.go
index c96b8a1..3475c3d 100644
--- a/main.go
+++ b/main.go
@@ -615,40 +615,6 @@ func main() {
 		proxyGemini(req, false, root, w, r, css, external)
 	}))
 
-	http.Handle("/x/", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if r.Method == "POST" {
-			r.ParseForm()
-			if q, ok := r.Form["q"]; !ok {
-				w.WriteHeader(http.StatusBadRequest)
-				w.Write([]byte("Bad request"))
-			} else {
-				w.Header().Add("Location", "?"+q[0])
-				w.WriteHeader(http.StatusFound)
-				w.Write([]byte("Redirecting"))
-			}
-			return
-		}
-
-		if r.Method != "GET" {
-			w.WriteHeader(http.StatusMethodNotAllowed)
-			w.Write([]byte("404 Not found"))
-			return
-		}
-
-		path := strings.SplitN(r.URL.Path, "/", 4)
-		if len(path) != 4 {
-			path = append(path, "")
-		}
-		req := gemini.Request{}
-		req.URL = &url.URL{}
-		req.URL.Scheme = "gemini"
-		req.URL.Host = path[2]
-		req.URL.Path = "/" + path[3]
-		req.URL.RawQuery = r.URL.RawQuery
-		log.Printf("%s (external) %s%s", r.Method, r.URL.Host, r.URL.Path)
-		proxyGemini(req, true, root, w, r, css, external)
-	}))
-
 	log.Printf("HTTP server listening on %s", bind)
 	log.Fatal(http.ListenAndServe(bind, nil))
 }
-- 
2.34.1

